### PR TITLE
Help: Release notes for 1.20.0

### PR DESCRIPTION
**Pull request by LesiaPavlenko**

### CODE REVIEW

File: help/deploy/releases/release-history.md [hunk 1]
```diff
@@ -20,155 +20,174 @@ position: 6 # float position is supported
...
-* (Bug) GROK-16309: Set tooltip dialog looks bad 
-* (Bug) GROK-16331: Core: Bar Chart: molecules are renderered incorrectly on resizing bar chart 
-* (Bug) GROK-16330: Core: Table view: Form styles with refresh button is broken 
 
+## 2024-07-23 Datagrok 1.20.0 release 
+ 
+### Breaking changes:
+ 
+* Lightweight Predictive modeling improvements.
+* File caching updates.
```
Comment (lbankurova) [inline help/deploy/releases/release-history.md:30]: @LesiaPavlenko How are these breaking changes? I may be wrong but the list here is different from what we have in the compatibility tool for the same

File: help/deploy/releases/release-history.md [hunk 2]
```diff
@@ -20,155 +20,174 @@ position: 6 # float position is supported
...
+* MlFlow registry integration.
+* Browse hierarchy updates.
+* Inputs harmonization.
+ 
+### Visualization and usability improvements:
+ 
+* GROK-15906: Harmonized layout editing for Data Query.
+* GROK-15908: Forms autosize.
+* GROK-15913: SaaS: Push notifications to stands about maintenance windows.
```
Comment (lbankurova) [inline help/deploy/releases/release-history.md:39]: @LesiaPavlenko Please remove anything SaaS-related

File: help/deploy/releases/release-history.md (hunk 1 above)
Comment (LesiaPavlenko) [reply help/deploy/releases/release-history.md:30]: Agree, I'll move them to "Improvements" part

File: help/deploy/releases/release-history.md (hunk 2 above)
Comment (LesiaPavlenko) [reply help/deploy/releases/release-history.md:39]: Done

**Comment by skalkin** [comment]
Overall, I find it pretty useless to an external person. Not sure what to do. Maybe we should drop these detailed release notes altogether.

**Review by skalkin — CHANGES_REQUESTED** [review]

File: help/deploy/releases/release-history.md [hunk 3]
```diff
@@ -20,155 +20,174 @@ position: 6 # float position is supported
...
+* GROK-15913: SaaS: Push notifications to stands about maintenance windows. 
+* GROK-15912: SaaS: Disable settings useless for client. 
+* GROK-15872: Browse: Functions menu is always expanded though it was closed before.
+* GROK-15322: Macromolecules updates.
+* GROK-15756: Diff Studio: Debugging mode.
+* GROK-15856: Toolbox: Layouts. Adding the loader for layouts.
+* GROK-14605: Context Pane: add visibility of the column names 
+* GROK-16206: Column and table metadata: register commonly used tags and expose them to UI.
+* GROK-15866: Projects: restyling dragging an entity into a project.
+* GROK-15009: Context Panel: make a tooltip explaining why the property is inactive. S
```
[the remark this answers is not in the document]
Comment (LesiaPavlenko) [reply help/deploy/releases/release-history.md]: I don't see that S letter in final version

File: help/deploy/releases/release-history.md [hunk 4]
```diff
@@ -20,155 +20,174 @@ position: 6 # float position is supported
...
+* GROK-15359: Changing logic for docking arrows. 
+* GROK-15427: Add an 'Add to favorites' command to objects. 
+* GROK-16247: Tooltips: ability to click a link in the tooltip. 
+* GROK-16274: Prevent auto running of functions in browse preview.
+* GROK-16278: Avoid sending extra data in files serialization. 
+* GROK-16279: Make browse less chatty.
```
[the remark this answers is not in the document]
Comment (LesiaPavlenko) [reply help/deploy/releases/release-history.md]: Already deleted this line in the final version

File: help/deploy/releases/release-history.md [hunk 5]
```diff
@@ -20,155 +20,174 @@ position: 6 # float position is supported
...
+### Visualization and usability improvements:
+ 
+* GROK-15906: Harmonized layout editing for Data Query.
+* GROK-15908: Forms autosize.
+* GROK-15913: SaaS: Push notifications to stands about maintenance windows. 
+* GROK-15912: SaaS: Disable settings useless for client. 
+* GROK-15872: Browse: Functions menu is always expanded though it was closed before.
+* GROK-15322: Macromolecules updates.
+* GROK-15756: Diff Studio: Debugging mode.
+* GROK-15856: Toolbox: Layouts. Adding the loader for layouts.
```
[the remark this answers is not in the document]
Comment (LesiaPavlenko) [reply help/deploy/releases/release-history.md]: Should I delete these 4?

File: help/deploy/releases/release-history.md [hunk 6]
```diff
@@ -20,155 +20,174 @@ position: 6 # float position is supported
...
+* Inputs harmonization.
+ 
+### Visualization and usability improvements:
+ 
+* GROK-15906: Harmonized layout editing for Data Query.
+* GROK-15908: Forms autosize.
+* GROK-15913: SaaS: Push notifications to stands about maintenance windows. 
+* GROK-15912: SaaS: Disable settings useless for client. 
+* GROK-15872: Browse: Functions menu is always expanded though it was closed before.
+* GROK-15322: Macromolecules updates.
```
[the remark this answers is not in the document]
Comment (LesiaPavlenko) [reply help/deploy/releases/release-history.md:40]: Deleted lines with SaaS.

File: help/deploy/releases/release-history.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (LesiaPavlenko) [reply help/deploy/releases/release-history.md]: Also felt strange when reviewed on the second time

File: help/deploy/releases/release-history.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (LesiaPavlenko) [reply help/deploy/releases/release-history.md]: Updated. Removed GROK-15872 and GROK-15856

**Pull request closed without merging**